server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/TestVariableEndDoesNotProduceNPE.java

```diff
@@ -50,7 +50,8 @@ void test() {
             "2",
             new Diagnostic(
                 new Range(),
-                "A period was assumed before \"END\".",
+                "Syntax error on 'END'",
+// TODO                "A period was assumed before \"END\".",
```

Comment:
this looks like a reasonable alternative
```suggestion
                "Syntax error on 'END'",
```